### PR TITLE
Update documents: LICENSE, README, CODE OF CONDUCT

### PR DESCRIPTION
**Pull request by Taowyoo**
Update: 
- LICENSE
- README
- CODE OF CONDUCT

### CODE REVIEW

File: key-attestation/java-example/README.md [hunk 1]
```diff
@@ -21,9 +21,38 @@ Online check
 Offline check
 - `verifyStatementFromJsonWithoutCrlCheck`: Verify given `KeyAttestationResponse` in a JSON file, assuming the last certificate in authority chain is correct ROOT certificate and skipping CRL checks.
 
+# Contributing
```
Comment (mzohreva) [inline key-attestation/java-example/README.md]: you don't need to repeat this everywhere, the root of the repo is sufficient

File: key-attestation/java-example/README.md [hunk 2]
```diff
@@ -21,9 +21,38 @@ Online check
...
+this project or the open source license(s) involved.
+
 # License
 
-This project is primarily distributed under the terms of the Apache License
-version 2.0 and the GNU General Public License version 2, see
-[LICENSE-APACHE](./LICENSE-APACHE) and [LICENSE-GPL](./LICENSE-GPL) for
-details.
\ No newline at end of file
+This project is primarily distributed under the terms of the Mozilla Public License (MPL) 2.0, see [LICENSE](./LICENSE) for details.
```
Comment (mzohreva) [inline key-attestation/java-example/README.md]: this is also redundant

File: key-attestation/java-example/LICENSE
Comment (mzohreva) [inline key-attestation/java-example/LICENSE:1]: I don't think this is necessary

File: key-attestation/java-example/CODE_OF_CONDUCT.md
Comment (mzohreva) [inline key-attestation/java-example/CODE_OF_CONDUCT.md:1]: I don't think this is necessary

File: key-attestation/java-example/README.md (hunk 1 above)
Comment (Taowyoo) [reply key-attestation/java-example/README.md]: I see, will delete those extra parts and files

Comment (Taowyoo) [reply key-attestation/java-example/README.md]: removed

File: key-attestation/java-example/LICENSE
Comment (Taowyoo) [reply key-attestation/java-example/LICENSE:1]: removed

File: key-attestation/java-example/CODE_OF_CONDUCT.md
Comment (Taowyoo) [reply key-attestation/java-example/CODE_OF_CONDUCT.md:1]: removed

**Review by mzohreva — APPROVED** [review]

**Review by xinyufort — APPROVED** [review]
:shipit:

**Pull request merged**